zerver/migrations/0398_tsvector_statistics.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.0.6 on 2022-07-18 23:22
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("zerver", "0397_remove_custom_field_values_for_deleted_options"),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql="ALTER TABLE zerver_message ALTER COLUMN search_tsvector SET STATISTICS 10000",
```

Comment:
It's probably worth copying a few sentences (or possibly more) from the commit message to a comment in this migration file.